apis/v1alpha2/gateway_types.go

```diff
@@ -18,32 +18,23 @@ package v1alpha2
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
 )
 
 // +genclient
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:categories=gateway-api,shortName=gtw
 // +kubebuilder:subresource:status
-// +kubebuilder:storageversion
+// +kubebuilder:deprecatedversion:warning="The v1alpha2 version of Gateway has been deprecated and will be removed in a future release of the API. Please upgrade to v1beta1."
 // +kubebuilder:printcolumn:name="Class",type=string,JSONPath=`.spec.gatewayClassName`
 // +kubebuilder:printcolumn:name="Address",type=string,JSONPath=`.status.addresses[*].value`
 // +kubebuilder:printcolumn:name="Ready",type=string,JSONPath=`.status.conditions[?(@.type=="Ready")].status`
```

Comment:
seems like this should be updated to `Accepted`? (and same for v1beta1)

Reply:
not sure what the compatibility promise is on printed columns/kubectl output

Reply:
I just submitted https://github.com/kubernetes-sigs/gateway-api/pull/1602 for this (switches to Programmed, but alternately could use Accepted if folks prefer that)

Reply:
ah yeah, Programmed is a closer replacement for Ready